### ISSUE TITLE
Add Security Consideration to avoid key reuse

### ISSUE THREAD

**Issue by msporny** [w3c/vc-data-integrity#169]
From the PING's review (https://github.com/w3cping/privacy-request/issues/120):
> Security section should make mention of key reuse considerations and the impact of reusing a key for multiple algorithms. While some combinations of algorithms have proofs that it's sufficiently secure to reuse keys for various algorithms many do not. This spec should discourage the reuse of keys whenever possible to prevent cryptography attacks that could reveal the private key.

/cc @kdenhartog

**Comment by msporny** [comment]
We should also reference this paper in the guidance: https://eprint.iacr.org/2021/509

**Comment by msporny** [comment]
Hrm, turns out that we do provide this guidance in the ECDSA Cryptosuite, so perhaps we should move that guidance to the main Data Integrity spec: https://www.w3.org/TR/2023/WD-vc-di-ecdsa-20230817/#key-management

**Comment by Wind4Greg** [comment]
I added that section, based on NIST and general recommendations. It is very general. So moving it to Data Integrity sounds good and avoids repetition in EdDSA, and future specs.

**Comment by kdenhartog** [comment]
I'd even suggest pushing up to the vc-data-model spec. While this issue is more commonly encountered in data-integrity formats from did:key it's an issue that re-applies across the entire space.

**Comment by msporny** [comment]
Agreed, this belongs in the vc-data-model as a general consideration regarding key re-use in any securing mechanism.
